pom.xml

```diff
@@ -42,7 +42,7 @@
         <maven.version>3.3.9</maven.version>
 
         <org.bouncycastle.version>1.63</org.bouncycastle.version>
-        <packager.version>0.16.1</packager.version>
+        <packager.version>0.16.2-SNAPSHOT</packager.version>
```

Comment:
Could you update this to 0.17.0, as the new version is now available on Maven Central.

Reply:
done